multiwallet.go

```diff
@@ -286,11 +291,11 @@ func (mw *MultiWallet) OpenWallets(startupPassphrase []byte) error {
 }
 
 func (mw *MultiWallet) AllWalletsAreWatchOnly() (bool, error) {
-	if len(mw.wallets) == 0 {
+	if len(mw.walletsReadCopy()) == 0 {
```

Comment:
Not sure there's an advantage to making and using a copy of the map here, the operation here doesn't take long at all. Should just do a lock and defer unlock at the top of this method and access the map directly.